src/main/java/com/atlauncher/evnt/listener/RelocalizationListener.java

```diff
@@ -17,6 +17,13 @@
  */
 package com.atlauncher.evnt.listener;
 
+/**
+ * Defines a class that can be called to change to a new language
```

Comment:
This isn't correct. It's more an interface to allow components to listen for changes to the language, not something that is called to change the language